nextflow_schema.json

```diff
@@ -741,6 +741,28 @@
                 }
             },
             "fa_icon": "fas fa-search"
+        },
+        "host_removal": {
+            "title": "Host Removal",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "run_hostremoval": {
+                    "type": "boolean",
+                    "description": "Turn on per-library creation pre-Adapter Removal FASTQ files without reads that mapped to reference (e.g. for public upload of privacy sensitive non-host data)",
+                    "help_text": "Create pre-Adapter Removal FASTQ files without reads that mapped to reference (e.g. for public upload of privacy sensitive non-host data)",
+                    "fa_icon": "fas fa-power-off"
+                },
+                "hostremoval_mode": {
+                    "type": "string",
+                    "default": "remove",
+                    "description": "Host removal mode. Remove mapped reads completely from FASTQ (remove) or just mask mapped reads sequence by N (replace).",
+                    "help_text": "Read removal mode. Remove mapped reads completely ('remove') or just replace mapped reads sequence by N ('replace')\n\n> Modifies extract_map_reads.py parameter: -m",
+                    "fa_icon": "fas fa-plane-slash"
```

Comment:
Missing the enum of options - in schema builder press 'cog' next to the entry and fill in the pipe separate d list

Reply:
done